21/day01/src/lib.rs

```diff
@@ -0,0 +1,47 @@
+fn parse_i32(s: &str) -> i32 {
+    s.parse::<i32>().unwrap()
+}
+
+fn add_three(a: &str, b: &str, c: &str) -> i32 {
+    parse_i32(a) + parse_i32(b) + parse_i32(c)
+}
+
+pub fn lib_main(input: &str) -> (i32, i32) {
```

Comment:
The lib_main ist too big. Separate it into 3 to 5 functions with descriptive names. Parameter names should also be descriptive.